src/extension/features/toolkit-reports/pages/outflow-over-time/utils.ts

```diff
@@ -0,0 +1,124 @@
+import lodash from 'lodash';
+import type { Moment } from 'moment';
+
+declare module 'moment' {
+  interface Moment {
+    toUTCMoment(): Moment;
+  }
+}
+
+interface YNABTransaction {
+  outflow?: number;
+  inflow?: number;
+  accountId: string;
+  transferAccountId: string | null;
+  date: Moment;
+  /** YYYY-mm format */
+  month: string;
+}
+
+export const filterTransactions = (
+  transactions: readonly YNABTransaction[],
+  filterOutAccounts: ReadonlySet<string>
+) => {
+  return transactions
+    .filter((transaction) => {
+      return (
+        transaction.outflow !== undefined &&
+        transaction.outflow !== 0 &&
+        (transaction.inflow === undefined || transaction.inflow === 0)
+      );
+    })
+    .filter((transactionsWithoutInflow) => {
```

Comment:
This filter is insufficiently precise in that it won't handle transfers inside splits correctly, but fixing that would be way more complicated than this for something a lot less common than ordinary transfers.